OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/files/FileChooserService.java

```diff
@@ -67,9 +67,11 @@ public static File getSelectedFile(String extension, String description, DialogT
         int returnVal = 0;
         switch (type) {
             case SAVE:
+                chooser.setDialogTitle("Save As");
```

Comment:
Sorry I misunderstood!  You're correct; there's no reason to keep those cases separate in the switch statement.  I'll merge them.